src/main/asciidoc/tasks/file-ingest/main.adoc

```diff
@@ -0,0 +1,102 @@
+
+=== Batch File Ingest
+
+In this demonstration, you will learn how to create a data processing application using http://projects.spring.io/spring-batch/[Spring Batch] which will then be run within http://cloud.spring.io/spring-cloud-dataflow/[Spring Cloud Data Flow].
+
+==== Prerequisites
+
+* A Running Data Flow Server
+include::{docs_dir}/local-server.adoc[]
+
+* A Running Data Flow Shell
+include::{docs_dir}/shell.adoc[]
+
+* Review the demo source code to understand how to create a data processing batch job application.
+
+==== Building and Running the Demo
+
+. Build the demo JAR
++
```

Comment:
Formatting looks good.  Can you add more content about what the app actually does? Walk through the code a bit more. See `streaming custom spring cloud stream processor`.

Reply:
sure - added an overview section that describes the application, and provides a brief overview of the key classes and what they do